### ISSUE TITLE
Result on CIFAR-10

### ISSUE THREAD

**Issue by ZitSaibok** [optimass/Maximally_Interfered_Retrieval#10]
Hi. Thanks for sharing your code.
I tested ER-MIR with 100 memories per class on Cifar10 several times, but get highest average accuracy of 0.454 +/- 0.018, lower than the accuracy of 0.476 +/- 0.011 in the original paper. 
I didn't change the code, and I ran the code as described in Scripts/ER_experiments.sh. Is there any configurations that should be modified to reproduce your results? Thank you very much!

**Comment by eugenium** [comment]
those +- overlap enough to make sense to me though it is a bit on the lower end. Keep in mind those report stderror the std of individual runs is a bit higher.  I would try rerunning with pytorch 1.1, might be some subtle differences regarding versions. Could also try different seeds

**Comment by eugenium** [comment]
Did you resolve this?

**Comment by pclucas14** [comment]
You can try with a lower learning rate (e.g. 0.05). I ran the code to reproduce the results for M=100 (with the new learning rate) for 10 runs and got `Final valid Accuracy: 0.489 +/- 0.017`. Hopefully this addresses your concern
